packages/@ionic/v1-toolkit/src/lib/gulp.ts

```diff
@@ -1,15 +1,49 @@
+'use strict';
+
 import { prettyPath } from '@ionic/cli-framework/utils/format';
 import * as chalk from 'chalk';
 import * as Debug from 'debug';
 import * as path from 'path';
+import * as fs from 'fs';
+import * as gulp from 'gulp';
+import * as undertaker from 'undertaker';
+import * as vm from "vm";
 
 import { timestamp } from './log';
+import {Context} from "vm";
+
+declare type GulpInst = typeof gulp & typeof undertaker;
```

Comment:
I believe the `declare` is unnecessary

```suggestion
type GulpInst = typeof gulp & typeof undertaker;
```